code-challenges.test.js

```diff
@@ -36,9 +119,65 @@ const letterE = "e"
 
 // b) Create the function that makes the test pass.
 
+// GREEN 
+// PSEUDOCODE
+// input: an array of words and a single letter 
+// output: an array of all the words containing that particular letter.
+
+// PROCESS:
+// create a function that takes 2 arguments, an array and a String,
+// create an empty string to hold your output array called result 
+// use a for loop to itereate of the lengthe of the words in the array 
+// use indexOf to check to the index of the letter you are looking for 
+// use push to push the word into your result array 
+// use toLowerCase to account to input casing 
+
+const wordsContainingLetter = (words, letter) => {
+    const result = []
+    for (let i = 0; i < words.length; i++) {
+      if (words[i].toLowerCase().indexOf(letter.toLowerCase()) !== -1) {
+        result.push(words[i])
+      }
+    }
+    return result
+  }
+  
+//   REFACTOR
+// instead of using a variable to create a new array use the spread operator which will create a new array 
+// use .filter to iterate ove the elements in th array  
+const wordsContainingLetter = (words, letter) => {
+    return [...words.filter(word => word.toLowerCase().includes(letter.toLowerCase()))]
+  }
```

Comment:
Great refactor! One step further, the `.filter()` method returns an array already, so no need to put that logic in an array with the spread operator!